resources/compass/values.yaml

```diff
@@ -143,18 +134,24 @@ global:
+
   istio:
     gateway:
-      name: "kyma-gateway"
-      namespace: "kyma-system"
+      name: compass-istio-gateway
+      namespace: compass-system
 
   database:
+    manageSecrets: true
```

Comment:
🤔 Should we really have `manageSecrets` turned on by default? Let's say someone takes kyma from the repo and tries to install it along with compass. All secrets will be missing from default installation, right? I think we should do this with overrides for our internal pipeline.

The same issue which I didn't think of previously is on `compass ` repo.

Reply:
I tried to keep the default bahavior here, so `manageSecrets: true` actually means let the helm chart "manage" the DB secrets by default.

Reply:
Also, for the default kyma installation with compass comes with the embedded DB enabled, so the managed Cloud SQL secret will not play a role here.

Reply:
Ah, okay, I get it. Makes sense 👍